### PR TITLE
Some small improvement

### PR DESCRIPTION
**Pull request by olvrng**
- [x] improve tree.Insert
- [x] add prependBytes
- [x] change slice pre-allocation from 100 to 128

### CODE REVIEW

File: pkg/storage/tree/tree.go [hunk 1]
```diff
@@ -108,6 +108,13 @@ func prepend(s []*treeNode, x *treeNode) []*treeNode {
 	return append(s, prev)
 }
 
+func prependBytes(s [][]byte, x []byte) [][]byte {
+	s = append(s, nil)
+	copy(s[1:], s)
+	s[0] = x
+	return s
+}
```
Comment (kolesnikovae) [inline pkg/storage/tree/tree.go:116]: I'm sure `prepend` can be done in the same way, without unnecessary checks, like:
```go
func prependTreeNode(s []*treeNode, x *treeNode) []*treeNode {
	s = append(s, nil)
	copy(s[1:], s)
	s[0] = x
	return s
}
```

**Review by kolesnikovae — COMMENTED** [review]
LGTM.

File: pkg/storage/tree/tree.go (hunk 1 above)
Comment (olvrng) [reply pkg/storage/tree/tree.go]: That's right. I added a commit for `prependTreeNode`.

**Review by petethepig — APPROVED** [review]
Looks good. Thanks @olvrng !

**Pull request merged**